ledger-tool/src/main.rs

```diff
@@ -39,6 +39,7 @@ fn main() {
         )
         .subcommand(SubCommand::with_name("print").about("Print the ledger"))
         .subcommand(SubCommand::with_name("json").about("Print the ledger in JSON format"))
```

Comment:
This command that predates your PR is awkward and puts us on a path where a user would expect a `*-tail` option for every future format. Instead, I'd expect there to be a `--format/-f=json` option. If that existed, then your PR would only add a `--tail` option and be used as `ledger-tool print --tail -f json`.  Pinging @rob-solana to see if he agrees.

Reply:
agreed